src/modules/Users/components/UsersList/UsersList.tsx

```diff
@@ -1,7 +1,305 @@
+
+                <th scope="col">Date Created</th>
+                <th scope="col">Action</th>
+              </tr>
+            </thead>
+            <tbody>
+              {userList.map((user: User) => (
+                <tr key={user.id}>
+                  <td> {user.userName}</td>
+                  <td>
+                    {user.isActivated ? (
+                      <span className="bg-success px-3 py-1 rounded-4 text-white fa-sm">
+                        active
+                      </span>
+                    ) : (
+                      <span className="bg-danger px-2 py-1 rounded-4 text-white fa-sm">
+                        Not Active
+                      </span>
+                    )}
+                  </td>
+                  <td>{user.phoneNumber}</td>
+                  <td>{user.email}</td>
+                  <td>{user.country}</td>
+
+                  <td>
+                    {user.imagePath == null ? (
+                      <img
+                        width={40}
+                        src={noPhoto}
+                        alt=""
+                        className="rounded-2"
+                      />
+                    ) : (
+                      <img
+                        width={40}
+                        src={`${BASE_IMG_URL}/${user.imagePath}`}
+                        alt=""
+                        className="rounded-2"
+                      />
+                    )}
+                  </td>
+                  <td>
+                    <div className="dropdown">
+                      <i className="fa-solid fa-ellipsis-vertical dropbtn"></i>
+                      <div className="dropdown">
+                        <div className="dropdown-content px-4 p-3 rounded-3 shadow">
+                          <span className="d-flex justify-content-between align-items-center">
+                            <i
+                              className="fa-solid fa-lock  pe-1 fa-sm"
+                              onClick={() => {
+                                blockUser(user.id);
+                              }}
+                            ></i>
+                            block
+                          </span>
+                          <span
+                            onClick={() => getUserById(user.id)}
+                            className=" d-flex justify-content-between align-items-center"
+                          >
+                            <i className="fa-regular   fa-eye pe-1 fa-sm"></i>
+                            View
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+
+          <div className="py-5 ps-4">
+            <nav>
+              <ul className="pagination">
+                {paginationNumber.map((p) => (
+                  <li className="page-item" onClick={() => getUserList(10, p)}>
```

Comment:
The onClick should be given to a button inside the list item, we should also remove the link.